classes/admin_setting_manage_metrics.php

```diff
@@ -155,7 +164,8 @@ public function output_html($data, $query='') {
                 $displayname,
                 $description,
                 $OUTPUT->render($editable),
-                $hideshow . $settingslink . $chartlink
+                $hideshow . $settingslink . $chartlink,
+                $backfilllink
```

Comment:
a super small general thing, with arrays it is nicer to add the trialing , on the last item so that next time we add an entry you don't get a diff which modifies an unrelated line